pkg/gvr/gvr.go

```diff
@@ -26,4 +26,10 @@ var (
 		Version:  "v2",
 		Resource: "servicemeshcontrolplanes",
 	}
+
+	JupyterhubApp = schema.GroupVersionResource{
+		Group:    "dashboard.opendatahub.io",
+		Version:  "v1",
+		Resource: "odhapplications",
+	}
```

Comment:
I think this should actually be `ODHApplication` instead. `jupyterhub` is a name of a concrete resource of this type, isn't it?

Reply:
https://github.com/opendatahub-io/odh-dashboard/blob/main/manifests/crd/odhapplications.dashboard.opendatahub.io.crd.yaml#L13 use the plural  
`Notice that resources are always lowercase, and by convention are the lowercase form of the Kind.`

Reply:
I meant exported variable name `JupyterhubApp` being `ODHApplication`, but generally speaking we can rename all of those to be plural

Reply:
aha,  jojo, now i got it :+1: